packages/react-native-bpk-component-text-input/src/BpkTextInput-test.common.js

```diff
@@ -178,6 +179,18 @@ const commonTests = () => {
 
       expect(testRenderer.toJSON()).toMatchSnapshot();
     });
+
+    it('should support theming', () => {
```

Comment:
This is not the a very good test as the style here is an animated value and only changes if the input is focused (and after the animation). We don't currently test any of this (what happens when it receives focus). I decided to follow this as adding a proper test will require a lot of work (and probably adding enzyme)

Reply:
Yeah that's fair enough. Perhaps we should have a small follow up card to do that. What do you think?

Reply:
Not sure how beneficial this would be to be honest, the cost/benefit of maintaining such a test doesn't seem good enough. Also we don't have many issues with this component as far as I know

Reply:
Alright cool 👍